tests/e2e/import.test.js

```diff
@@ -0,0 +1,199 @@
+  let page;
+
+  const passwordErrorLabel = '#options-root > div > div.MuiContainer-root.MuiContainer-maxWidthSm > div > div.MuiGrid-root.makeStyles-passwordError-4.MuiGrid-item.MuiGrid-grid-xs-12 > p';
+
+  const badSeedphrase = 'sadf adsf adfdfasd adfad adfafd sdfsd sdfsdf sdfds sdfd sdf sfsfs sdfadsf';
+
+  const getButtonCollection = async (page) => {
+    return await page.$$('button');
+  }
+
+  const getImportButton = async (page) => {
+    const buttonCollection = await getButtonCollection(page);
+    return buttonCollection[0];
+  };
+
+  const getCreateButton = async (page) => {
+    const buttonCollection = await getButtonCollection(page);
+    return buttonCollection[1];
+  };
+
+  beforeAll(async () => {
+    chromeBrowser = await setupChrome();
+  });
+  
+  beforeEach(async () => {
+    page = await chromeBrowser.newPage();
+    await page.goto(chromeData.optionsUrl);
+  });
+
+  afterEach(async () => {
+    await page.close();
+  });
+
+  afterAll(async () => {
+    await chromeBrowser.close();
+  });
+
+  describe('Import', () => {
+    beforeEach(async () => {
+      const importButton = await utils.getButtonWithIndex(page, 0);
+      await importButton.click();
+    });
+
+    test('Fails on incorrect seedphrase', async () => {
+      const importInputElement = await utils.getElement(page, 'textarea');
+      await importInputElement.type(badSeedphrase);
+
+      const submitImport = await utils.getButtonWithIndex(page, 0);
+      await submitImport.click();
+
+      const updatedImportBtn = await utils.getButtonWithIndex(page, 0);
+      const importButtonClassName = await (await updatedImportBtn.getProperty('className')).jsonValue();
+      const importButtonClassNameArray = importButtonClassName.split(' ');
+
+      expect(importButtonClassNameArray).toEqual(expect.arrayContaining(['Mui-disabled']));
+    });
+
+    test('Fails on missmatched passwords', async () => {
+      const importInputElement = await utils.getElement(page, 'textarea');
+      await importInputElement.type(secrets.seedphrase);
+
+      const submitImport = await utils.getButtonWithIndex(page, 0);
+      await submitImport.click();
+
+      await utils.waitForRender(500);
+      const [passwordInput, confirmPasswordInput] = await utils.getInputs(page);
+      await passwordInput.type('TestPassword123');
+      await confirmPasswordInput.type('MissMatchedPassword');
+
+      const submitPassword = await utils.getButtonWithIndex(page, 0);
+      await submitPassword.click();
+
+      await page.waitForSelector(passwordErrorLabel);
+      const labelErrorElement = await page.$(passwordErrorLabel)
+      const value = await page.evaluate(el => el.textContent, labelErrorElement);
+
+      expect(value).toBe('The passwords gotta match, smh!');
+    });
+
+    test('Fails on password shorter than 12 characters', async () => {
+      const importInputElement = await utils.getElement(page, 'textarea');
+      await importInputElement.type(secrets.seedphrase);
```

Comment:
where are these secrets stored?

Reply:
This secrets can be set-up in `tests/setup/jestSetup.js`, we should probably make them sync with `ENV` variables so we can do testing via GH actions